packages/conventional-recommended-bump/README.md

```diff
@@ -97,6 +97,11 @@ Type: `string` Possible values: `'major'`, `'minor'` and `'patch'`
 
 The value of what it should release as.
 
+##### tagPrefix
+
+Type: `string`
+
+Specify a prefix for the git tag that will be taken into account during the comparison.
```

Comment:
Could you provide an example of how this works in practice? Reading the code below, it seems obvious, but from just this one line of documentation, I'm not sure what this option does for me, or how I can use it.

Reply:
isn't it preferable to style your tags with a suffix instead? something like `1.0.0-beta.1`, etc? By doing a suffix, it also seems to lend itself towards using npm's interface to increment the same release type:

```
command-line example:

$ semver 1.2.3 -i prerelease --preid beta
1.2.4-beta.0
Which then can be used to increment further:

$ semver 1.2.4-beta.0 -i prerelease
1.2.4-beta.1
```

https://docs.npmjs.com/misc/semver#prerelease-identifiers

Reply:
@hellatan sure, but sometimes you don't have a choice how your tags are built :) (especially in enterprise environment)

Reply:
> isn't it preferable to style your tags with a suffix instead? something like 1.0.0-beta.1, etc?

@hellatan, for the semantic version number, that is correct.

A prefix is not part of a valid semantic version (Though `v` was part of an earlier version of the [SemVer spec](https://semver.org/)).

However, prefixes are used quite often in the context of mono-repositories where individual packages within the mono-repository need to be tagged with version `1.0.0`. The only way to distinguish between the `1.0.0` tag for `package-a` versus `package-b` is to prefix the tag with the name of the package.

When `package-a` or `pakcage-b` is published, they are only published using the semantic version value of the tag, excluding the package prefix.

So if I were to increment the version for `package-a`, I would fetch all tags with the prefix `package-a`, remove the prefix, and then sort based on semantic versioning. Then I could pass the latest value to `semver` as you demonstrated.

Reply:
right...which we do at work for our mono-repo as well 😄. The project i'm working is a single repo so i'm not concerned with the prefix so it didn't really come to mind.